src/util/createInstantSearch.js

```diff
@@ -12,13 +12,16 @@ See more info here: https://community.algolia.com/vue-instantsearch/components/I
   );
 
 export const createInstantSearch = ({ ssr }) => ({
-  name: 'AisInstantSearch',
+  name: `AisInstantSearch${ssr ? 'Ssr' : ''}`,
```

Comment:
Instead of concatenating letters to the name, it's often better to rewrite the whole name like so:

```suggestion
  name: ssr ? 'AisInstantSearchSsr' : 'AisInstantSearch',
```

It makes it easier to read and to find when searching in the source code.